src/components/patch/PatchContainer.test.tsx

```diff
@@ -44,17 +56,30 @@ describe("PatchContainer", () => {
     expect(failedBox).toHaveLength(1);
     expect(variant.state("name")).toBe("Ubuntu 16.04");
     expect(variant.state("statusCount")).toEqual({ success: 2, failed: 1 });
-    expect(variant.state("sortedStatus")).toEqual([{"count": 1, "status": "failed"}, {"count": 2, "status": "success"}]);
+    expect(variant.state("sortedStatus")).toEqual([{ "count": 1, "status": "failed" }, { "count": 2, "status": "success" }]);
+  })
+
+  it("check that expanded state persists on re-render", () => {
+    expect(wrapper.state("expandedPatches")).toEqual({});
+    const patch = wrapper.findWhere(node => node.key() === "5d1385720305b932b1d50d01");
```

Comment:
This actually finds the wrapper around the patch component. Would it be easier to find the patch component itself so you don't have to look at its children?

Reply:
Fixed